### ISSUE TITLE
BUG: df.values[0] doesn't return an integer type

### ISSUE THREAD

**Issue by WuTianyi321** [xprobe-inc/xorbits#319]
### Describe the bug
`df.values[0]` doesn't return an integer type

### To Reproduce
To help us to reproduce this bug, please provide information below:

Python: 3.8.8 (tags/v3.8.8:024d805, Feb 19 2021, 13:18:16) [MSC v.1928 64 bit (AMD64)] on win32
xorbits: 0.2.1

Python code:
```import xorbits
xorbits.init()
from xorbits import pandas as pd
t=pd.DataFrame({"ttt":range(10,20)})
m=t["ttt"].head(1).values[0]
for i in range(m):
    print(i)
```

Stack:
```
      3 t=pd.DataFrame({"ttt":range(10,20)})
      4 m=t["ttt"].head(1).values[0]
----> 5 for i in range(m):
      6     print(i)

TypeError: 'DataRef' object cannot be interpreted as an integer
```

**Comment by qianduoduo0904** [comment]
Could you please translate the title and content into English, so that everyone from all over world can understand. Thank you in advance.

**Comment by Zhou1213CN** [comment]
Take

**Comment by qianduoduo0904** [comment]
This issue has been addressed on main branch, and will be included in upcoming Xorbits v0.3.0 in a few weeks.
